src/node.cpp

```diff
@@ -81,7 +81,7 @@ int main(int argc, char **argv)
 
       size_t idx = 0;
       for (auto [angle_milli_deg, distance_mm, signal_strength] : scan.samples) {
```

Comment:
```suggestion
      for (auto [angle_milli_deg, distance_cm, signal_strength] : scan.samples) {
```
I would rename the variable `distance_mm` to `distance_cm` both in this line and the next. Then the code is semantically correct too (not just functionally) since the output of the scanner is not millimeter but centimeter.